### PR TITLE
fix(list view): Fix the work item list display

### PR DESCRIPTION
**Pull request by nimishamukherjee**
When a new work item is added or edited, the list needs to be displayed based on the applied filters.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1960-fabric8-planner.int.rdu2c.fabric8.io

**Comment by nimishamukherjee** [comment]
[test]

**Comment by vikram-raj** [comment]
@nimishamukherjee Is it for list view only? Because its not working with board view.

**Comment by nimishamukherjee** [comment]
@vikram-raj - yes this is for list view only

**Comment by nimishamukherjee** [comment]
[test]

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1960-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@nimishamukherjee snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1960-fabric8-planner.int.rdu2c.fabric8.io

File: src/app/components/planner-list/planner-list.component.ts [hunk 1]
```diff
@@ -504,8 +504,15 @@ export class PlannerListComponent implements OnInit, AfterViewInit, DoCheck, OnD
 
     this.eventListeners.push(
       this.workItemService.addWIObservable.subscribe(item => {
-        //move the update list logic
-      }));
+        this.loadWorkItems();
```
Comment (sanbornsen) [inline src/app/components/planner-list/planner-list.component.ts]: This should add the work item on top of the list i.e. this.workItems

File: src/app/components/planner-list/planner-list.component.ts [hunk 2]
```diff
@@ -504,8 +504,15 @@ export class PlannerListComponent implements OnInit, AfterViewInit, DoCheck, OnD
...
+      })
+    );
+
+    this.eventListeners.push(
+      this.workItemService.editWIObservable.subscribe(item => {
+        this.loadWorkItems();
```
Comment (sanbornsen) [inline src/app/components/planner-list/planner-list.component.ts]: This should find the item in the list (i.e. this.workItems) and then update that particular item.

**Pull request merged**